Wikibase.php

```diff
@@ -108,6 +108,20 @@
 		// hardcoded internal handling
 		'html',
 	];
+	$wgWBRepoSettings['entityDataCachePaths'] = function() [
```

Comment:
Should be a { instead of [ at the end. Same to close

Reply:
I so much thought of that, I was just starring at it lol